builtin/providers/postgresql/provider.go

```diff
@@ -35,6 +35,12 @@ func Provider() terraform.ResourceProvider {
 				DefaultFunc: schema.EnvDefaultFunc("POSTGRESQL_PASSWORD", nil),
 				Description: "Password for postgresql server connection",
 			},
+			"sslmode": &schema.Schema{
```

Comment:
According to [this documentation](http://www.postgresql.org/docs/9.5/static/libpq-ssl.html), it is defined as `sslmode`. 
By using `connStr`, it looks like `postgresql://localhost:2345/postgres?sslmode=require`
By using lib [pq library](https://godoc.org/github.com/lib/pq), it looks like `db, err := sql.Open("postgres", "user=postgres dbname=postgres sslmode=require")`